### PR TITLE
Issue#151 merge miecd to master

### PR DESCRIPTION
**Pull request by FrankApiyo**
## What changes are introduced by this PR

- Add `*/*/node_modules/` to `.gitignore` file

- Move most files from project's root directory to `clients/miecd`

- Delete `tslint.json`, `LICENSE`, `.prettierrc` and `README.md`

For: #151

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -43,14 +43,12 @@ jobs:
             # fallback to using the latest cache if no exact match is found
```
Comment (FrankApiyo) [inline .circleci/config.yml]: This file has been modified in order to be able to run tests for `miecd` for now; Most of this file will be replaced with whatever is in master.

**Comment by FrankApiyo** [comment]
The `config.yml`  file has been modified in order to be able to run tests for miecd for now; Most of this file will be replaced with whatever is in master.

**Comment by p-netm** [comment]
@FrankApiyo  the circle ci build is having trouble with saving and restoring this project's cache

File: clients/miecd/src/helpers/tests/utils.test.tsx [hunk 2]
```diff
@@ -0,0 +1,239 @@
+import * as gatekeeper from "@onaio/gatekeeper";
```
Comment (moshthepitt) [inline clients/miecd/src/helpers/tests/utils.test.tsx]: I think it would have been best for this not to be included in the PR.  Lessons for the future.

Comment (FrankApiyo) [reply clients/miecd/src/helpers/tests/utils.test.tsx]: cool.

**Comment by FrankApiyo** [comment]
> @FrankApiyo the circle ci build is having trouble with saving and restoring this project's cache

@p-netm, because we do not have the correct jest configs on this branch, we will have this problem here. Once we merge to master where jest is configured correctly, this will be solved

**Comment by FrankApiyo** [comment]
> Looks good. I think you should add a README that is specific for this project. You might need to work with @hmakandi on what it should say.

This PR is currently awaiting a `README` that is specific to the **MIECD** project from @hmakandi

File: .gitignore [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 # dependencies
 /node_modules
+*/*/node_modules/
```
Comment (cKellyDesign) [inline .gitignore]: @FrankApiyo let's consolidate these two lines into `node_modules/` to [ignore `node_modules` directories anywhere in the repo](https://stackoverflow.com/questions/29820791/git-ignore-node-modules-folder-everywhere)

File: LICENSE [hunk 4]
```diff
@@ -1,201 +0,0 @@
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
```
Comment (cKellyDesign) [inline LICENSE]: @FrankApiyo why was the License removed?

Comment (FrankApiyo) [reply LICENSE]: @cKellyDesign, the intention for removing the `LICENCE` file and many other files from the `miecd_vietnam` branch root directory is to ensure that they are replaced with similar files in the master branch when the merging is eventually done; so this `LICENCE` file will be replaces by the `LICENCE` file in the master branch(https://github.com/OpenSRP/opensrp-web/blob/master/LICENSE)

Comment (FrankApiyo) [reply LICENSE]: @cKellyDesign thanks for taking time to review this.

File: .gitignore (hunk 3 above)
Comment (FrankApiyo) [reply .gitignore]: done!

**Pull request merged**